superset-frontend/src/views/dashboardList/DashboardList.tsx

```diff
@@ -173,7 +174,7 @@ class DashboardList extends React.PureComponent<Props, State> {
         },
       }: any) => (
         <span className="no-wrap">
-          {published ? <i className="fa fa-check" /> : ''}
+          {published ? <i className="fa fa-check text-muted" /> : ''}
```

Comment:
is this color supposed to match the other Icons? it doesn't currently. Maybe replace this with `<Icon name="check' />`

Reply:
+1 this. I think we should move to use our own svg/Icon component and finally remove Font Awesome. otherwise, i always feel confused which icon to use :(